api/v1alpha1/paasconfig_types.go

```diff
@@ -301,14 +308,16 @@ func (config PaasConfigSpec) CapabilityK8sName(capability string) (as types.Name
 
 type PaasConfigStatus struct {
 	// Conditions of this resource
-	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"`
+	// +kubebuilder:validation:Optional
```

Comment:
As represented in the generated manifest, this has no effect nor is needed. Please don't add this here.

Reply:
For consistency, I have reverted all // +kubebuilder:validation:Optional tags to omitempty flags.